src/architecture/utilitiesSelfCheck/_UnitTest/test_swigDeprecated.py

```diff
@@ -0,0 +1,64 @@
+#
```

Comment:
This PR should have release notes stating that this new depreciation capability is being added.